contents/blog/running-group-targeted-ab-tests.md

```diff
@@ -0,0 +1,106 @@
+
+## Combining user-targeted and group-targeted experiments to reduce risk
+
+To improve the quality of insights from group-targeted experiments and address the risks mentioned above, you may want to consider running an experiment with your change at a user-level before running it at a group-level. This can help you in the following ways:
+
+### 1. Save time for changes that don't have an impact
+
+As mentioned, lower statistical power in group-targeted experiments means you need to run your experiments longer. However, a change that doesn't impact behavior at a user-level won't, in all likelihood, impact behavior at a group-level either.
+
+To rule this out, consider running a user-targeted experiment first in order to see if there is any impact on individual user behavior (where it should be quicker to gain results from because of higher statistical power) before running a group-targeted experiment.
+
+### 2. Gain user-level insights and identify randomization risks
+
+By running a user-targeted experiment first, you'll gain insight into how individual users interact with your change and what characteristics they may possess. This will then give you an understanding of what results you can expect from a group-targeted experiment.
+
+This can also help you identify randomization risks. For example, let's say a B2B SaaS app runs a user-targeted experiment and notices that users in large companies are more likely to interact with their new feature. In this case, when they run a group-targeted experiment, they can ensure to filter out small and medium-sized companies in their experiment in order to obtain more accurate results.
+
+## How to set up group-targeted experiments in PostHog
+
+First, ensure you have [set up a group](docs/product-analytics/group-analytics#getting-started) in PostHog. Next, go to your PostHog dashboard and create a [new experiment](docs/experiments/manual#creating-an-experiment).
+
+In the New Experiment screen, select the Participant type. By default, it will show `Persons` (i.e. a user-level experiment). Click on the drop-down and select your new group.
+
+![Screenshot of setting the participant type in an experiment](../images/blog/running-group-targeted-ab-tests/participant-type.png)
+
+Next, set your experiment goal and secondary metrics as usual. 
+
+A handy feature especially useful for group-level experiments is the Minimum Acceptable Improvement calculator at the bottom of the page. Since group-level experiments have lower statistical power, this will recommend a time for how long you should run your experiment in order to see results.
+
+![Screenshot of the Minimum Acceptable Improvement calculator](../images/blog/running-group-targeted-ab-tests/minimum-acceptable-improvement.png)
+
+
```

Comment:
This article needs a summary to wrap it up – basically imagine a TL;DR. Something like:

"To summarize, remember these x points when running group-targeted experiments:

1. XYZ
2. XYZ
3. XYZ

To learn more about running product experiments, read our guides to `links`.